src/http/server.clj

```diff
@@ -293,6 +294,21 @@
          (log/infof "\"%s %s HTTP/1.1\" %s %s" method'  uri status len))
        response))))
 
+(defn auth-prompt []
+  (let [username (do (println "Username: ") (read-line))
+        password (do (println "Password: ") (.readPassword (System/console)))]
+    (str username ":" password)))
+
+(defn maybe-inject-auth
+  [auth]
```

Comment:
I would personally prefer `partial` to be consistent with other functions.

Reply:
OK - I will make this change.